bin/main.rb

```diff
@@ -0,0 +1,48 @@
+
```

Comment:
Let's make `main.rb` an executable file.

Adding `#!/usr/bin/env ruby` to this line will make `bin/main.rb` run as `ruby bin/main.rb` would.
```suggestion
#!/usr/bin/env ruby
# frozen_string_literal: true
```

### Please do this before merging.